cmp/grid/GridModel.js

```diff
@@ -310,12 +318,21 @@ export class GridModel {
             showCellFocus
         });
 
-        this.colChooserModel = enableColChooser ? this.createChooserModel() : null;
+        this.colChooserModel = this.parseChooserModel(colChooserModel);
         this.selModel = this.parseSelModel(selModel);
         this.persistenceModel = persistWith ? new GridPersistenceModel(this, persistWith) : null;
         this.experimental = this.parseExperimental(experimental);
     }
 
+    confirmRestoreDefaults() {
```

Comment:
wonder if a better pattern here would be to just add an additional flag to the existing restoreDefaults method of the form `requireConfirm= false`?

the name `confirmRestoreDefaults` seems a bit ambiguous to me